### PR TITLE
Add simplecov

### PR DESCRIPTION
**Pull request by dks17**
I would like add test coverage.

- Let me know Codeclimate test reporter id for `mongoid-locker`.
- Should we update `mongoid` minor versions in the `Gemfile` to latest (i.g. `~> 7.0` to `~> 7.4`)?
- Should we drop support of `mongoid` version `2` and `3`? They are not covered with tests.
- Should we drop testing with `rbx-2` (I suppose it is Rubinius)?

### CODE REVIEW

**Comment by dblock** [comment]
> Let me know Codeclimate test reporter id for mongoid-locker.

You should be able to set this up via https://codeclimate.com/github/repos/new because you have access to this repo, no? Or does it need repo admin access in which case I am happy to help.

> Should we update mongoid minor versions in the Gemfile to latest (i.g. ~> 7.0 to ~> 7.4)?

I think we should.

> Should we drop support of mongoid version 2 and 3? They are not covered with tests.

Yes if we're not actually testing them. Or actually test them. I think it's time :)

> Should we drop testing with rbx-2 (I suppose it is Rubinius)?

If it's broken, otherwise who cares?

File: Gemfile [hunk 1]
```diff
@@ -25,6 +25,8 @@ group :development do
   gem 'rb-fsevent', '~> 0.9.1'
 end
 
+gem 'simplecov', require: false, group: :test
```
Comment (dblock) [inline Gemfile]: Move it into the group below?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,6 +1,11 @@
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 
+require 'simplecov'
+SimpleCov.start 'rails' do
```
Comment (dblock) [inline spec/spec_helper.rb]: I don't know much about simplecov, but what's up with Rails here?

Comment (dks17) [reply spec/spec_helper.rb]: It is the most suitable preset. I have already restricted it.
https://github.com/colszowka/simplecov/blob/master/lib/simplecov/profiles/rails.rb

**Comment by dks17** [comment]
I have only access to my fork. Or I just want to make you a little more happier. I need your help @dblock .

**Comment by dblock** [comment]
I made you collaborator -> admin on this repo. Try again?

File: .travis.yml [hunk 3]
```diff
@@ -9,28 +9,37 @@ before_install:
   - gem update --system
   - gem install bundler
 
+env:
+  global:
+    - CC_TEST_REPORTER_ID=4462a28198e28892091f0afb6d13417325549906f26182911786e8ea8735c471
```
Comment (dblock) [inline .travis.yml]: Should we move this into CI's configuration (in Travis, I can add it or I believe you can too) and remove it from the checked in file?

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,7 @@
 ### 1.0.1 (Next)
 
 * Your contribution here.
+* [#60](https://github.com/mongoid/mongoid-locker/pull/60): Drop support for `mongoid` version `2` and `3`, Add SimpleCov - [@dks17](https://github.com/dks17).
```
Comment (dblock) [inline CHANGELOG.md]: Split these up in two entries.

**Comment by dblock** [comment]
This looks good. Thanks! Feel free to consider the ENV change/make it and to merge after.

File: .travis.yml (hunk 3 above)
Comment (dks17) [reply .travis.yml]: Nice suggestion. Added into Travis. Thanks.

**Pull request merged**